### PR TITLE
Fix interactables, update scene hierarchy to be more clear for end-user

### PR DESCRIPTION
**Pull request by RobCharisma**
- Fixed interactable meta functions
    - Metadata sent from graph functions included multiple interactables at once, while the interactable functions only handled 1 object at a time, so they didn't work. Now they handle all the objects.
- Scene hierarchy updated
    - Renamed NPCs and added their IDs to match the plug n play NPCs
    - Renamed some scene objects to be more clear and eye-catching for the end-user
    - Removed MainThreadConsumer, this is to be merged alongside this [SDK PR](https://github.com/charisma-ai/charisma-sdk-unity/pull/27)

### CODE REVIEW

File: Samples~/Scripts/Playthrough/Metadata/AllowInteractableFunction.cs [hunk 1]
```diff
@@ -9,16 +9,20 @@ internal class AllowInteractableFunction : MetadataFunction
 
         public override void Execute(string metadataValue)
         {
-            var entity = Dependencies.PlaythroughEntities.GetEntityByName(metadataValue);
+            string[] subjectTargets = metadataValue.Split(',');
```
Comment (RaduDumitriu-Dev) [inline Samples~/Scripts/Playthrough/Metadata/AllowInteractableFunction.cs:12]: awesome, think we have this in some of our other PnP projects but I've forgot to port it over. Great call.

File: Samples~/Scenes/PlugNPlaySample.unity
Comment (RaduDumitriu-Dev) [inline Samples~/Scenes/PlugNPlaySample.unity:1]: small thing, but playthrough instance still has the Playthrough parameters in the sample scene. We don't normally include these in our packages partly out of safety and to encourage people to sign up on the platform.

Comment (RaduDumitriu-Dev) [reply Samples~/Scenes/PlugNPlaySample.unity:1]: Hierarchy organisation looks good, a lot nicer to sift thru.

**Review by RaduDumitriu-Dev — CHANGES_REQUESTED** [review]
Looks good! Only thing would be clearing the scene of the playthrough parameters and I can approve

File: Samples~/Scenes/PlugNPlaySample.unity
Comment (RobCharisma) [reply Samples~/Scenes/PlugNPlaySample.unity:1]: Should be fixed now!

**Comment by RaduDumitriu-Dev** [comment]
Ran another test on this - will need to update the Charisma.Audio namespace references in the PlaythroughInstances from /~samples/. That should be the only other change needed

**Review by RaduDumitriu-Dev — APPROVED** [review]

**Pull request merged**